cmd/kustomize/main.go

```diff
@@ -43,6 +44,14 @@ func New(testCtx *test.TestCommandContext, kustomizeCtx *KustomizeContext) *cobr
 		# Test the kustomize build from github
 		datree kustomize test https://github.com/kubernetes-sigs/kustomize.git/examples/helloWorld?ref=v1.0.6
 		`),
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
this solves the problem! can we just extract this check to avoid code duplication?

Reply:
I made the required changes and hope that changes are based on golang code standards.

Reply:
fixing the fail stages of pipelines